doc/install/raspbian-jessie.md

```diff
@@ -0,0 +1,24 @@
+# Installing cjdns on raspbian jessie
+
+This is a short guide how to setup a raspbian jessie cjdns box.
+
+## Install packages
+
+```bash
+sudo apt install nodejs build-essential git-svn
```

Comment:
`git` is a meta-package including various GUI clients and the actual Git (named `git-svn`).